src/storage/hub.ts

```diff
@@ -72,20 +74,7 @@ export async function deleteFromGaiaHub(
     }
   )
   if (!response.ok) {
-    let responseMsg = ''
-    try {
-      responseMsg = await response.text()
-    } catch (error) {
-      Logger.debug(`Error getting bad http response text: ${error}`)
-    }
-    const errorMsg = 'Error deleting file from Gaia hub: '
-      + `${response.status} ${response.statusText}: ${responseMsg}`
-    Logger.error(errorMsg)
-    if (response.status === 404) {
-      throw new FileNotFound(errorMsg)
-    } else {
-      throw new Error(errorMsg)
-    }
+    throw getBlockstackErrorFromResponse(response, 'Error deleting file from Gaia hub.')
```

Comment:
I think this needs an `await`

Reply:
That may explain some of the failing tests, a few `UnhandledPromiseRejectionWarning: Unhandled promise rejection` warnings in the logs.